1906-12-17.xml

```diff
@@ -0,0 +1,2158 @@
+                                <cell>about <date when="1906-12-14">14 December</date></cell>
+                            </row>
+                            <row>
+                                <cell><name>Sachsen</name></cell>
+                                <cell><measure quantity="5026" unit="ton">5026</measure> Tons</cell>
+                                <cell>about <date when="1905-12-28">28 December</date></cell>
+                            </row>
+                            <row>
+                                <cell><name>Gneisenau</name></cell>
+                                <cell><measure quantity="8881" unit="ton">8881</measure> Tons</cell>
+                                <cell>about <date when="1906-01-11">11 January</date></cell>
+                            </row>
+                            <row>
+                                <cell><name>Pr. Regt. Luitpold</name></cell>
+                                <cell><measure quantity="6288" unit="ton">6288</measure> Tons</cell>
+                                <cell>about <date when="1905-01-25">25 January</date></cell>
+                            </row>
+                            <row>
+                                <cell cols="3">OUTWARD: for CHINA and JAPAN via SUEZ, ADEN, COLOMBO, PENANG,
+                                    SINGAPORE.</cell>
+                            </row>
+                            <row>
+                                <cell><name>Prinz E. Friedrich</name></cell>
+                                <cell><measure unit="ton" quantity="8965">8965</measure> Tons</cell>
+                                <cell>about <date when="1906-12-24">24 December</date></cell>
+                            </row>
+                            <row>
+                                <cell><name>Preussen</name></cell>
+                                <cell><measure unit="ton" quantity="5295">5295</measure> Tons</cell>
+                                <cell>about <date when="1907-01-07">7 January</date></cell>
+                            </row>
+                            <row>
+                                <cell><name>Roon</name></cell>
+                                <cell><measure unit="ton" quantity="8022">8022</measure> Tons</cell>
+                                <cell>about <date when="1907-01-21">21 January</date></cell>
+                            </row>
+                            <row>
+                                <cell cols="3">For AUSTRALIA via SUEZ, ADEN, COLOMBO.</cell>
+                            </row>
+                            <row>
+                                <cell><name>Seydlitz</name></cell>
+                                <cell><measure unit="ton" quantity="7943">7943</measure> Tons</cell>
+                                <cell>about <date when="1906-12-16">16 December</date></cell>
+                            </row>
+                            <row>
+                                <cell><name>Z
```

Comment:
should be `<div type="section" element="local">`, then all of the stories in the section can be items with own headlines.